models/api.model.js

```diff
@@ -0,0 +1,9 @@
+const {getApi}=require('./api.controller')
+const express = require('express')
```

Comment:
Not sure what's happening in this model? What is it's purpose? it looks like you are creating an exporting another express server here but we already have one in app.js? 👀